src/popup/pages/NotFound.vue

```diff
@@ -1,23 +1,95 @@
 <template>
   <div class="not-found">
-    {{ $t('pages.notFound.notFound') }}
-    <BtnMain :to="{ name: 'index' }">
-      {{ $t('pages.notFound.initialPage') }}
+    <img
+      class="image"
+      :src="Image"
+    >
+    <div class="title">
+      {{ $t('pages.notFound.notFound') }}
+    </div>
+    <div class="description">
+      {{ $t('pages.notFound.description') }}
+    </div>
+    <BtnMain
+      class="button"
```

Comment:
Actually you can use a `btn-main` class to refer that element and not set additional class. But that is something to be defined
https://github.com/aeternity/superhero-wallet/issues/1676

Reply:
I'm opting for leaving the additional classes :)
It's more clear for the component internal code and makes the further refactors easier.
But yes, the conversation on standardising this is in progress. For now I would suggest to leave the decision to the code author.

Reply:
I suggest leaving this solution